django_project/trail/utils.py

```diff
@@ -0,0 +1,21 @@
+# import hashlib
```

Comment:
Never add a file with all commented code :P It is better to not have anything.

Reply:
Typically we can remember stuff by looking at git history. If you need to tell future you something, then use comments written in English rather than Code.